### PR TITLE
Fix typography style page style bug

### PR DESCRIPTION
**Pull request by Honglin-Lu**
- Fix the style bug of typography style page, since we override the Mui table theme.
- Small modification of the page based on the latest figma design.
image

### CODE REVIEW

**Review by wenfeng-xing — APPROVED** [review]

**Review by wenfeng-xing — APPROVED** [review]

File: docs/src/design-system/typography/TextStylesPage.tsx [hunk 1]
```diff
@@ -34,37 +34,38 @@ const rows = [
...
           sx={{
+            [`& .${tableCellClasses.root}`]: {
+              p: '0',
+              verticalAlign: 'middle',
+            },
             width: '45.75rem',
-            margin: '2.4375rem auto 22rem',
+            margin: '2.5rem auto 22rem',
+            borderBottom: 1,
+            borderColor: palette.divider,
```
Comment (tprince232) [inline docs/src/design-system/typography/TextStylesPage.tsx:67]: I'd expect the `Table` by default with out theme applied would already have a borderBottom and borderColor as specified, no?

Here is the default table with our theme:
image

File: docs/src/design-system/typography/TextStylesPage.tsx [hunk 2]
```diff
@@ -74,7 +75,7 @@ const TextStylesPage: React.FC = () => {
                     key={column}
                     align="left"
                     sx={{
-                      ...tableCellStyles(palette),
+                      ...tableCellStyles,
```
Comment (tprince232) [inline docs/src/design-system/typography/TextStylesPage.tsx:78]: You know, you can do this just fine with `sx={tableCellStyles}`, but if you prefer the way you have it with the spread, that's fine - it does have the advantage of being able to easily add on other styles if needed.

File: docs/src/design-system/typography/TextStylesPage.tsx (hunk 1 above)
Comment (Honglin-Lu) [reply docs/src/design-system/typography/TextStylesPage.tsx:67]: Yes, by default table already have a borderBottom and borderColor, but the last row of borderBottom of the tableBody was 
 also removed by our theme, which was implemented in the `MuiTableBody.ts` I think. So to be consistent with the design, I add a borderBottom specially for the table.
image

Comment (tprince232) [reply docs/src/design-system/typography/TextStylesPage.tsx:67]: Okay just looked more closely at the designs - in this case, I think the design is wrong for being inconsistent, but anyways this is fine.

File: docs/src/design-system/typography/TextStylesPage.tsx (hunk 2 above)
Comment (Honglin-Lu) [reply docs/src/design-system/typography/TextStylesPage.tsx:78]: Thanks for pointing it out!

**Review by tprince232 — APPROVED** [review]

**Pull request merged**